### PR TITLE
[NUI] Add some newly added APIs for API10

### PR DESCRIPTION
**Pull request by bshsqa**
### API Changes ###

 - ACR:

### CODE REVIEW

**Comment by Seoyeon2Kim** [comment]
lol

**Comment by dongsug-song** [comment]
@bshsqa 
Hello,
could you check the build error https://github.com/Samsung/TizenFX/actions/runs/5783363613/job/15671990266?pr=5454
and please add the description.
thank you.

File: src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionIndex/MotionTransformIndex.cs [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Copy constructor.
+        /// </summary>
+        /// <param name="motionTransformIndex">Source object to copy.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public MotionTransformIndex(BlendShapeIndex motionTransformIndex) : this(Interop.MotionIndex.NewMotionTransformIndex(MotionTransformIndex.getCPtr(motionTransformIndex)), true)
```
Comment (hinohie) [inline src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionIndex/MotionTransformIndex.cs:136]:
```suggestion
        public MotionTransformIndex(MotionTransformIndex motionTransformIndex) : this(Interop.MotionIndex.NewMotionTransformIndex(MotionTransformIndex.getCPtr(motionTransformIndex)), true)
```
Please change this  typo error

**Review by hinohie — APPROVED** [review]
Dali patch merged

**Pull request merged**